### ISSUE TITLE
Closing out expired/outdated/replaced feeds

### ISSUE THREAD

**Issue by mldickens** [TransitFeeds/TransitFeeds-Public#394]
One example, Connecticut Transit has 7 GTFS feeds listed, because there used to be 7 different feeds. These were consolidated into two feeds. Only Hartford and Shore Line East are still valid.

Is there a way we can mark these old feeds as depreciated in order to 1) stop the system from trying to download data and throwing fail errors and 2) show people using the API that this feed is no longer up-to-date?

**Comment by HendX** [comment]
I agree this would be useful - what did you have in mind for the API? So like, there's a deprecated checkbox in admin, then that value would be included in the feed too?

**Comment by HendX** [comment]
@mldickens - also, I can't find your email address. Would you mind shooting over an email to me? support@transitfeeds.com is fine

**Comment by canales** [comment]
@caywood this is an important one. We currently have no way to do this from the admin UI.

Even deleting feeds which were created twice because of human error, like:
https://transitfeeds.com/p/ferrotramviaria-spa/843
http://transitfeeds.com/p/ferrotramviaria/412
https://transitfeeds.com/p/dtpm-santiago
https://transitfeeds.com/p/dtpm-santiago-santiago

**Comment by caywood** [comment]
I propose we create an admin mode checkbox to deprecate a feed. 

Optionally, you can add a feed ID. For example if you deprecate 412 in the example above, you add 843 to this field.

![screen shot 2019-03-01 at 6 06 46 pm](https://user-images.githubusercontent.com/11506462/53671606-302e4d80-3c4d-11e9-9ab6-72e683252448.png)
